crt_portal/crt_portal/settings.py

```diff
@@ -158,11 +156,12 @@
     STATIC_URL = f'https://{AWS_S3_CUSTOM_DOMAIN}/{AWS_LOCATION}/'
     STATICFILES_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
     DEFAULT_FILE_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
-
 else:
     STATIC_URL = '/static/'
 
-STATIC_ROOT = os.path.join(BASE_DIR, "static")
+STATICFILES_DIRS = ( os.path.join(BASE_DIR, 'static'), )
+MEDIA_URL = 'media/'
+STATIC_ROOT = os.path.join(BASE_DIR, "staticfiles")
```

Comment:
@LindsayYoung Would you mind adding one or two lines of code comments about how STATIC_ROOT and STATICFILES_DIRS work? Trying to wrap my mind around how Django static assets are working here. Thank you!

Reply:
I did add the one liners- Static assets are confusing I added some one liners. There is a link to the docs above this section. 

I would recommend watching our former college, Jacob: 
https://www.youtube.com/watch?v=E613X3RBegI

Reply:
Thanks!